en/ui-system/components/engine/index.md

```diff
@@ -1,49 +1,64 @@
-# UI Architecture
+# UI Description
 
-The UI rendering is based on a tree structure. The **Canvas** node (a node with `Canvas component`) is the root of UI. **All ui nodes must be under the Canvas node to be rendered**. The `UITransform` is the necessary condition that each UI node must have for a click or alignment strategy, etc. to take effect.
+## Getting Started with UI
 
-The UI supports 2D & 3D camera hybrid sorting. Control the sorting by switching the rendering mode and adjusting the `priority` property on the component.
+The difference between defining UI and 2D rendering objects in the engine mainly lies in adaptation and interaction. All UI needs to be under Canvas node to make adaptation behavior, and Canvas component itself inherits from `RenderRoot2D` component, so it can also be used as an entry point for data collection.
```

Comment:
```suggestion
The difference between defining UI and 2D rendering objects in the engine mainly lies in adaptation and interaction. All UI needs to be under a Canvas node to make adaptation behavior, and the Canvas component itself inherits from `RenderRoot2D` component, it can also be used as an entry point for data collection.
```